src/Framework/CustomServiceAwareTrait.php

```diff
@@ -13,7 +13,7 @@ trait CustomServiceAwareTrait
 
     public function __call(string $name, array $arguments = []): CustomServiceInterface
     {
-        $resolvableType = ltrim($name, 'get');
+        $resolvableType = ucfirst(ltrim($name, 'get'));
```

Comment:
Don't forget this method is not working properly, we should use a regex_replace like \^get\ to empty?